en-US/ps-menu-help.xml

```diff
@@ -0,0 +1,178 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file autogenerated? if so, please add a command to readme, otherwise I think it will be too hard to maintain

Reply:
I have added information to the ReadMe.